src/main/kotlin/com/cognifide/gradle/aem/instance/InstanceSync.kt

```diff
@@ -108,7 +140,27 @@ class InstanceSync(aem: AemExtension, instance: Instance) : InstanceHttpClient(a
         return response
     }
 
-    fun downloadPackage(remotePath: String, targetFile: File, retry: Retry = aem.retry()) {
+    fun downloadPackage(definition: VltDefinition.() -> Unit): File {
```

Comment:
creates empty package on the fly then uploads and build its on AEM, then downloads and removes remote package; 
effectively previously empty package will be replaced with built one